.github/workflows/tests.yml

```diff
@@ -17,9 +24,8 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install -r requirements.txt
-          pip install -r requirements/requirements.test.txt
-          pip install -e .
+          pip install -e .[test]
+          pip install gunicorn
```

Comment:
`.[test]` has other stuff we need - we could install `.[prod]` as well, but ultimately we shouldn't as we don't really need to run the API under `gunicorn` for testing - we only do this because Flask built-in server doesn't like being daemonized.

`requirements.txt` is based on `.[prod]` for exactly this reason :)